pom.xml

```diff
@@ -162,11 +162,11 @@ flexible messaging model and an intuitive client API.</description>
     <docker-maven.version>0.42.1</docker-maven.version>
     <docker.verbose>true</docker.verbose>
     <typetools.version>0.5.0</typetools.version>
-    <protobuf3.version>3.19.6</protobuf3.version>
+    <protobuf3.version>3.22.3</protobuf3.version>
```

Comment:
Protobuf 3.22.0 introduces a breaking change. It requires all the sub-project, which depend on the bookkeeper to upgrade Protobuf to 3.22.0+ https://github.com/apache/bookkeeper/pull/4001. 
We use [lightproto](https://github.com/splunk/lightproto) to compile Protobuf code to Java code and the lightproto uses Protobuf 3.13.0. Do we need to upgrade lightproto's Protobuf version first before upgrading Pulsar's Protobuf version?